### PR TITLE
Make SFTP target directory configurable

### PR DESCRIPTION
**Pull request by NickGrantX**
# Motivation and Context
SFTP target directory needs to be easily configurable between different environments (e.g. CI, WL, BL, Prod etc)

# What has changed
Use k8s configmap parameter to configure the target SFTP directory.

# How to test?
Build an environment with branch `sftp-target-directory-config` on `census-rm-terraform` and `census-rm-kubernates`.

# Links
Trello: https://trello.com/c/jTMYmVyo

# Screenshots (if appropriate):

### CODE REVIEW

**Comment by NickGrantX** [comment]
[Make SFTP directory configurable for batch runner (3)](https://trello.com/c/jTMYmVyo/860-make-sftp-directory-configurable-for-batch-runner-3)

File: run_gke.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ kubectl run acceptance-tests -it --command --rm --quiet --generator=run-pod/v1 \
     --env=SFTP_USERNAME=$(kubectl get secret sftp-ssh-credentials -o=jsonpath="{.data.username}" | base64 --decode) \
     --env=SFTP_KEY=$(kubectl get secret sftp-ssh-credentials -o=jsonpath="{.data.private-key}") \
     --env=SFTP_PASSPHRASE=$(kubectl get secret sftp-ssh-credentials -o=jsonpath="{.data.passphrase}" | base64 --decode) \
-    --env=SFTP_DIR=${PROJECT_NAME_CONFIG}/upload/print_service/ \
+    --env=SFTP_DIR=${SFTP_TARGET_DIRECTORY}/print_service/ \
```
Comment (jcox-dev) [inline run_gke.sh:50]: Does this `/print_service/` still need to be present?

File: tasks/kubectl-run-acceptance-tests.yml [hunk 2]
```diff
@@ -40,7 +42,7 @@ run:
       --env=SFTP_USERNAME=$(kubectl get secret sftp-ssh-credentials -o=jsonpath="{.data.username}" | base64 --decode) \
       --env=SFTP_KEY=$(kubectl get secret sftp-ssh-credentials -o=jsonpath="{.data.private-key}") \
       --env=SFTP_PASSPHRASE=$(kubectl get secret sftp-ssh-credentials -o=jsonpath="{.data.passphrase}" | base64 --decode) \
-      --env=SFTP_DIR=${GCP_PROJECT_NAME}/upload/print_service/ \
+      --env=SFTP_DIR=${SFTP_TARGET_DIRECTORY}/print_service/ \
```
Comment (jcox-dev) [inline tasks/kubectl-run-acceptance-tests.yml:45]: Likewise, does this `/print_service/` still need to be present?

**Comment by jcox-dev** [comment]
`BATCH_RUNNER_CONFIG=https://raw.githubusercontent.com/ONSdigital/census-rm-qid-batch-runner/sftp-target-directory-config/qid-batch-runner.yml`

**Review by jcox-dev — APPROVED** [review]

**Comment by jcox-dev** [comment]
Yas! to the QID branch config 👍

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**